.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -5,6 +5,6 @@
 <!-- describe a summary of the change, add any additional motivation and context as needed -->
 
 # Verification steps
-<!-- describe verification steps with sufficient level of detail -->
-<!-- take into consideration upgrade scenario, RHMI 2.x, etc. -->
+<!-- Add thorough verification steps with sufficient level of detail for those without context to verify the change-->
+<!-- AND Add thorough upgrade verification OR include a reason as to why it is not required -->
```

Comment:
Do you want steps here?
```suggestion
<!-- AND Add thorough upgrade verification steps OR include a reason as to why it is not required -->
```
The rest of it looks great to me.

Reply:
Updated.